config/features.yml

```diff
@@ -365,9 +365,6 @@ features:
     actor_type: user
     description: Enables users to submit FSR forms for VHA and VBA debts
     enable_in_development: true
-  communication_preferences:
-    actor_type: user
-    description: Allow user to access backend communication_preferences API
```

Comment:
Forgotten flipper! [communication_preferences](https://api.va.gov/flipper/features/communication_preferences) flipper has been removed so this is no longer needed.